taca/nanopore/instrument_transfer.py

```diff
@@ -25,7 +25,9 @@ def main(args):
 
     logging.info("Starting script...")
 
-    run_pattern = re.compile("\d{8}_\d{4}_[A-Za-z0-9]+_[A-Za-z0-9]+_[A-Za-z0-9]+")
+    run_pattern = re.compile(
```

Comment:
I would add a comment here to explain the purpose of the [1-2][A-H] , even though it's easy to find int he git blame. It will quite likely be changed back sometime in the future

Reply:
[24856cd](https://github.com/SciLifeLab/TACA/pull/401/commits/24856cd973f79f2112e2a1b69035eeeecbca26e6)